lib/me-keyword.resource

```diff
@@ -0,0 +1,22 @@
+*** Settings ***
+Library     BuiltIn
+Library     Process
+
+
+*** Keywords ***
+Check ME State
+    [Documentation]    Keyword Checks The ME PCI Register Offset 0x40 To Find Current ME State, Returns:
+    ...    -Disabled(soft) For 3
+    ...    -Disabled(HAP) for 2
+    ...    -Enabled for 0
+
+    ${out}=    Execute Command In Terminal    setpci -s 16.0 40.L
+    ${char}=    Evaluate    ${out}[3]
+    IF    '${char}' == '3'
+        ${result}=    Evaluate    'Disabled(soft)'
+    ELSE IF    '${char}' == '2'
+        ${result}=    Evaluate    'Disabled(HAP)'
```

Comment:
a/a
```suggestion
        ${result}=    Evaluate    'Disabled (HAP)'
```

Reply:
changes included in: https://github.com/Dasharo/open-source-firmware-validation/pull/375/commits/e8738cef4b6e89953cb45972e3df68a60965bd21